backend/lib-shared-types/src/entity/site_api/site_entity.rs

```diff
@@ -13,6 +13,7 @@ pub struct SiteEntity {
     pub pages: String,
     pub created_at: DateTime<Utc>,
     pub updated_at: DateTime<Utc>,
+    pub content_updated_at: i64,
```

Comment:
I changed this to `i64` because it was getting complicated to make sure the data in the sqlite3 db matched the query update_key. It's also more efficient, and I don't there there are benefits to using a date string.

The reason we use a date/string for the previous `update_key` is that we already had `updated_at` in that format.